play.py

```diff
@@ -29,17 +26,16 @@ def get_action(self, obs):
         manual_override = self.real_controller.LeftBumper == 1
 
         if not manual_override:
-
             ## Look
             vec = resize_image(obs)
-
+            vec = np.expand_dims(vec, axis=0)  # expand dimensions for predict, it wants (1,66,200,3) not (66, 200, 3)
             ## Think
-            joystick = \
-              model.y.eval(session=self.sess, feed_dict={model.x: [vec], model.keep_prob: 1.0})[0]
+            joystick = self.model.predict(vec, batch_size=1)  # proces input image 1 by 1
+            joystick = np.squeeze(joystick)  # remove the extra dimension [[0, 1 ]] -> [0, 1], not sure how acquired
```

Comment:
can you re-write this like it was above?

```python
self.model.predict(vec, batch_size=1)[0]
```

I think its cleaner that way. We can also remove the comments.

The extra dimension is there because sometimes you want to execute in batches, like during training or if you were doing a classification problem you could do it all at once.